### PR TITLE
Script/Professor Putricide: Fix Tear Gas

### LINKED ISSUE #7972: [ICC]Professor Putricide - Tear Gas Don't Work
Hi.At Professor Putricide in tranzition phase tear gas don't work.I searched a fix but i didn't find it/

rev: https://github.com/TrinityCore/TrinityCore/commit/74707a08d3d63dc66e7b6943431c91f50258b0f9
DB: TDB_335.49.
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/63807-icc-professor-putricide-tear-gas-don-t-work?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Aokromes**
[//]: # (***************************)
[//]: # (** FILL IN THIS TEMPLATE **)
[//]: # (***************************)

**Changes proposed:**

-  Fixes tear gas
**Target branch(es):** 3.3.5/master

- [x] 3.3.5

**Issues addressed:** Closes #  (insert issue tracker number)
Closes #7972

**Tests performed:** (Does it build, tested in-game, etc.)

[//]: # (**DEPRECATION NOTICE** Instead of creating new PRs to the (old) 6.x branch, make sure the target branch is **master** instead of 6.x.)
[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request (if not enabled already).)
[//]: # (**NOTE** If this PR __only__ contains SQL files, open a new issue instead and post or link the SQL there.)
[//]: # (**SUGGESTION** When adding new SQL files, name them 9999_99_99_99_db_name.sql to reduce the chance of possible merge conflicts.)

By someone who deleted account
Closes #7972

### CODE REVIEW

**Comment by Shauren** [comment]
wtf?

**Comment by Krudor** [comment]
:D

File: src/server/scripts/Northrend/IcecrownCitadel/boss_professor_putricide.cpp [hunk 1]
```diff
@@ -606,6 +607,8 @@ class boss_professor_putricide : public CreatureScript
                             Talk(EMOTE_UNSTABLE_EXPERIMENT);
                             DoCast(me, SPELL_UNSTABLE_EXPERIMENT);
                             events.ScheduleEvent(EVENT_UNSTABLE_EXPERIMENT, urand(35000, 40000));
+                            WhileCasting = true;
+                            events.ScheduleEvent(EVENT_WHILE_CASTING, 2500); // SPELL_UNSTABLE_EXPERIMENT Cast Time
                             break;
```
Comment (joschiwald) [inline src/server/scripts/Northrend/IcecrownCitadel/boss_professor_putricide.cpp]: try to replace this break with return, and it should work without these changes

Comment (Keader) [inline src/server/scripts/Northrend/IcecrownCitadel/boss_professor_putricide.cpp]: dont work here joschi

**Pull request closed without merging**